src/ResourceManagement/OperationalInsights/OperationalInsights.Tests/Helpers/LinkedServiceUndoHandler.cs

```diff
@@ -0,0 +1,83 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using Hyak.Common;
+using Microsoft.Azure.Management.OperationalInsights;
+using Microsoft.Azure.Management.OperationalInsights.Models;
+using Microsoft.Azure.Test;
+
+namespace OperationalInsights.Tests.Helpers
+{
+    /// <summary>
+    /// Undo Handler for linked service operations
+    /// </summary>
+    public class LinkedServiceUndoHandler : ComplexTypedOperationUndoHandler<ILinkedServiceOperations, OperationalInsightsManagementClient>
+    {
+        /// <summary>
+        /// Look up the undo operation for the given linked service operation
+        /// </summary>
+        /// <param name="client">The IWorkspaceOperations instance used for the operation</param>
+        /// <param name="method">The name of the workspace method</param>
+        /// <param name="parameters">The parameters passed to the operation</param>
+        /// <param name="undoAction">The undo action for the given operation, if any</param>
+        /// <returns>True if an undo operation is found, otherwise false</returns>
+        protected override bool DoLookup(IServiceOperations<OperationalInsightsManagementClient> client, string method, IDictionary<string, object> parameters, out Action undoAction)
```

Comment:
>IDictionary<string, object> parameters, [](start = 120, length = 39)

Nit: Consider splitting long parameter list into multiple lines, makes it easy to code review in github UI. Also helps in general from readability point of view.